### PR TITLE
Add note about arguments object to seek and destroy challenge

### LINKED ISSUE #11271: Seek and Destroy. From the beginning the function destroyer has one argument to pass. But when we call this function we pass 3 values in it.
Challenge [Seek and Destroy](https://www.freecodecamp.com/challenges/seek-and-destroy) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 6.1; WOW64; rv:49.0) Gecko/20100101 Firefox/49.0</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:

``` javascript

function destroyer(arr,val1,val2,val3) {
  // Remove all the values
  //alert('sum=' + val1 +'another val2='+val2);
  function identic(value){

    return (value !== val1 );

  }
  function identic1(value){

    return (value !== val2 );

  }
  function identic2(value){

    return (value !== val3 );

  }


   arr = arr.filter(identic);
   arr = arr.filter(identic1);
  arr = arr.filter(identic2);
  return arr;

}

destroyer([1, 2, 3, 1, 2, 3], 2, 3);

```

### PR DESCRIPTION
**Pull request by systimotic**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #11271

#### Description

This PR adds a note about having to use the arguments object to this challenge, as per the issue.
The link to the MDN page about the arguments object was already in place, which is why I didn't add it to the message.

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/basic-bonfires.json [hunk 1]
```diff
@@ -671,6 +671,7 @@
       "title": "Seek and Destroy",
       "description": [
         "You will be provided with an initial array (the first argument in the destroyer function), followed by one or more arguments. Remove all elements from the initial array that are of the same value as these arguments.",
+        "<strong>Note:</strong> You have to use the <code>arguments</code> object.",
```
Comment (erictleung) [inline seed/challenges/01-front-end-development-certification/basic-bonfires.json]: I think we usually have a `<br>` after the Note and no colon like in [this line](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/1f105f38610bd442fcad7ebabbc82406843ee5f6/seed/challenges/01-front-end-development-certification/basic-javascript.json#L505).

**Review by erictleung — CHANGES_REQUESTED** [review]
Almost good. But I think to be consistent with preexisting code, there should be a line break and no colon.

**Comment by systimotic** [comment]
Ah, thank you for bringing this up. I was thinking about it, but then forgot about it.
I'm happy to chanhe this tomorrow when I get back to my laptop.
The reason I went with this format is https://github.com/FreeCodeCamp/FreeCodeCamp/pull/11805 It might be a good idea to agree on a common format for this.



On Tue, Nov 29, 2016 at 11:17 PM +0100, "Eric Leung" <notifications@github.com<mailto:notifications@github.com>> wrote:


@erictleung requested changes on this pull request.

Almost good. But I think to be consistent with preexisting code, there should be a line break and no colon.

________________________________

In seed/challenges/01-front-end-development-certification/basic-bonfires.json<https://github.com/FreeCodeCamp/FreeCodeCamp/pull/11867#pullrequestreview-10651543>:

> @@ -671,6 +671,7 @@
       "title": "Seek and Destroy",
       "description": [
         "You will be provided with an initial array (the first argument in the destroyer function), followed by one or more arguments. Remove all elements from the initial array that are of the same value as these arguments.",
+        "<strong>Note:</strong> You have to use the <code>arguments</code> object.",


I think we usually have a <br> after the Note and no colon like in this line<https://github.com/FreeCodeCamp/FreeCodeCamp/blob/1f105f38610bd442fcad7ebabbc82406843ee5f6/seed/challenges/01-front-end-development-certification/basic-javascript.json#L505>.

-

**Comment by erictleung** [comment]
@systimotic ah I see. When I made one of these "Notes" a while back, I was told to go with the styling I mentioned. I agree that we should have some consistent way of formatting the notes.

**Comment by HKuz** [comment]
For what it's worth, I've been using the &lt;strong&gt;Note&lt;/strong&gt;&lt;br&gt; formatting in the curriculum expansion challenges. (I think the first example in the existing code I found did it that way). I have that and a few other minor formatting nits in a list of things that we'll need to make consistent across all challenges - making the change in existing challenges will probably be one of the last things I do for the QA process 😄.

**Comment by Bouncey** [comment]
## LGTM :+1: 

Thanks for contributing :tada:

**Pull request merged**